include/rviz_visual_tools/rviz_visual_tools.h

```diff
@@ -352,6 +352,8 @@ class RvizVisualTools
                         const rviz_visual_tools::colors color = BLUE);
   bool publishRectangle(const geometry_msgs::Point &point1, const geometry_msgs::Point &point2, 
                         const rviz_visual_tools::colors color = BLUE);
+  bool publishRectangle(const geometry_msgs::Pose &pose, const double l, const double w, const double h,
```

Comment:
Can you spell out l,w,h in the var names? Esp. since you didn't add documentation above the function

How do you feel about calling length "depth" ?